pkg/branchcuts/bumper/release-controller-config-bumper.go

```diff
@@ -0,0 +1,212 @@
+		return releaseConfig, err
+	}
+	if err := ReplaceWithNextVersionInPlace(&releaseConfig.MirrorPrefix, b.mm.Major); err != nil {
+		return releaseConfig, err
+	}
+	if err := ReplaceWithNextVersionInPlace(&releaseConfig.Name, b.mm.Major); err != nil {
+		return releaseConfig, err
+	}
+	if err := ReplaceWithNextVersionInPlace(&releaseConfig.OverrideCLIImage, b.mm.Major); err != nil {
+		return releaseConfig, err
+	}
+
+	if err := bumpReleaseCheck(releaseConfig, b.mm.Major); err != nil {
+		return releaseConfig, err
+	}
+	if err := bumpReleasePublish(releaseConfig, b.mm.Major); err != nil {
+		return releaseConfig, err
+	}
+	if err := bumpReleaseVerification(releaseConfig, b.mm.Major); err != nil {
+		return releaseConfig, err
+	}
+
+	setMapIfNil(&releaseConfig.Check, map[string]ReleaseCheck{})
+	setMapIfNil(&releaseConfig.Verify, map[string]ReleaseVerification{})
+	setMapIfNil(&releaseConfig.Publish, map[string]ReleasePublish{})
+
+	return releaseConfig, nil
+}
+
+func bumpReleaseCheck(releaseConfig *ReleaseConfig, currentMajor int) error {
+	if releaseConfig.Check == nil {
+		return nil
+	}
+	check := make(map[string]ReleaseCheck)
+	for k, v := range releaseConfig.Check {
+		if v.ConsistentImages != nil {
+			if err := ReplaceWithNextVersionInPlace(&v.ConsistentImages.Parent, currentMajor); err != nil {
+				return err
+			}
+		}
+		check[k] = v
+	}
+	releaseConfig.Check = check
+	return nil
+}
+
+func bumpReleasePublish(releaseConfig *ReleaseConfig, currentMajor int) error {
+	if releaseConfig.Publish == nil {
+		return nil
+	}
+	publish := make(map[string]ReleasePublish)
+	for k, v := range releaseConfig.Publish {
+		if v.VerifyBugs != nil && v.VerifyBugs.PreviousReleaseTag != nil {
+			if err := ReplaceWithNextVersionInPlace(&v.VerifyBugs.PreviousReleaseTag.Name, currentMajor); err != nil {
+				return err
+			}
+			if err := ReplaceWithNextVersionInPlace(&v.VerifyBugs.PreviousReleaseTag.Tag, currentMajor); err != nil {
+				return err
+			}
+		}
+		if v.ImageStreamRef != nil {
+			if err := ReplaceWithNextVersionInPlace(&v.ImageStreamRef.Name, currentMajor); err != nil {
+				return err
+			}
+		}
+		if v.TagRef != nil {
+			if err := ReplaceWithNextVersionInPlace(&v.TagRef.Name, currentMajor); err != nil {
+				return err
+			}
+		}
+		publish[k] = v
+	}
+	releaseConfig.Publish = publish
+	return nil
+}
+
+func bumpReleaseVerification(releaseConfig *ReleaseConfig, currentMajor int) error {
+	if releaseConfig.Verify == nil {
+		return nil
+	}
+	verify := make(map[string]ReleaseVerification)
+	for k, v := range releaseConfig.Verify {
+		if v.ProwJob != nil {
+			if err := ReplaceWithNextVersionInPlace(&v.ProwJob.Name, currentMajor); err != nil {
+				return err
+			}
+		}
+		newKey, err := ReplaceWithNextVersion(k, currentMajor)
+		if err != nil {
+			return err
+		}
+		verify[newKey] = v
+	}
+	releaseConfig.Verify = verify
+	return nil
+}
+
+func (b *ReleaseControllerConfigBumper) Marshall(releaseConfig *ReleaseConfig,
```

Comment:
nit: the name of this method is not correct.

Reply:
Yeah you're right, even if `marshalling` is allowed as well, check [here](https://en.wikipedia.org/wiki/Marshalling_(computer_science)). Changing that would mean to modify files that are not strictly related with the purpose of this PR, I'm planning to open another one.